platform/DirHandle.h

```diff
@@ -35,12 +35,19 @@ namespace mbed {
  *  but only a subset needs to be provided.
  *
  *  If a FileSystemLike class defines the opendir method, then the
- *  directories of an object of that type can be accessed by
- *  DIR *d = opendir("/example/directory") (or opendir("/example")
- *  to open the root of the filesystem), and then using readdir(d) etc.
+ *  directories of an object of that type can be accessed by either:
+ *  @code
+ *  DIR *d  = opendir("/example/directory");
+ *  @endcode
+ *  or
+ *  @code
+ *  DIR *d = opendir("/example");
+ *  @endcode
+ *  to open the root of the filesystem and then using readdir,
+ *  closedir, etc with the directory pointer.
```

Comment:
Duno why, but this sentence feels like a both a run-on sentence, and like it's worded weirdly.

Reply:
Yeah sort of choppily rewrote that segment, one second.

Reply:
Is this referring to the FileSystem class or any file system? If the latter, we should make it two words.